.github/workflows/rust-checks.yml

```diff
@@ -39,8 +39,6 @@ jobs:
       
       - name: Cache dependencies
         uses: Swatinem/rust-cache@v2
-        with:
-          key: base 
```

Comment:
Removed since before we had many `--features` -- now that we use dockerized ere, we can use a single key for Rust cache.